### PR TITLE
Documentation for Travel Allowance API 

### PR DESCRIPTION
**Pull request by UdayKumarNettem**

### CODE REVIEW

File: src/api-reference/expense/travelallowance/travelallowance-days.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+---
+title: TravelAllowance Days
+layout: reference
+---
+
+This Travel Allowance API fetches all the fixed allowance days in an expense report. Clients using this API can determine whether an allowance is a full day or partial day allowance.
```
Comment (sarraloew) [inline src/api-reference/expense/travelallowance/travelallowance-days.md]: Change "in an expense report" to "in the specified expense report"

File: src/api-reference/expense/travelallowance/travelallowance-days.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
+---
+title: TravelAllowance Days
```
Comment (sarraloew) [inline src/api-reference/expense/travelallowance/travelallowance-days.md]: Put a space between Travel and Allowance

File: src/api-reference/expense/travelallowance/travelallowance-days.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+layout: reference
+---
+
+This Travel Allowance API fetches all the fixed allowance days in an expense report. Clients using this API can determine whether an allowance is a full day or partial day allowance.
+
+> **Limitations**: This API is available only for Finland customers.
```
Comment (sarraloew) [inline src/api-reference/expense/travelallowance/travelallowance-days.md:8]: Is this also only available in the EMEA data center?

File: src/api-reference/expense/travelallowance/travelallowance-days.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+Name|Description|Endpoint
+---|---|---|
+travelallowance.allowancedays.read|View the allowance days in an expense report|GET
+
+## <a name="access-token-usage"></a>Access Token Usage
+
+This API will work with Company access token only. A valid Company `access token` is needed to use this API.
+
+## Retrieve a Company Access Token
```
Comment (sarraloew) [inline src/api-reference/expense/travelallowance/travelallowance-days.md]: This heading should be H3

File: src/api-reference/expense/travelallowance/travelallowance-days.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+
+Using Company-level access token is required for creating an integration with this API. This method will allow your integration to get fixed allowance day details of an expense report.  
+
+For clients connecting to this API to build a custom integration you will receive client credentials and information on how to generate your company access token or company refresh token from the Web services resource assigned to assist you with Authentication.
+
+Before making requests to this API, you must [obtain an access token from the Authentication API](/api-reference/authentication/getting-started.html).
+
+The response will include an `access_token` field, which contains your access token. For subsequent calls, you will need to include this access token in the `Authorization` header of your calls. You will need this in order to get fixed allowance day details.
+
```
Comment (sarraloew) [inline src/api-reference/expense/travelallowance/travelallowance-days.md:48]: Move the Get Details of Travel Allowance Days heading up here, as H2.

File: src/api-reference/expense/travelallowance/travelallowance-days.md [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+```shell
+https://{datacenterURI}/api/v4/travelallowance/allowancedays/companyUUID/{companyUUID}/contexts/
+{Context}/{ContextId}
+```
+
+##### Parameters
+
+Name|Type|Format|Description
+---|---|---|---
+`companyUUID`|`uuid`|-|**Required** Note required when applicable.
```
Comment (sarraloew) [inline src/api-reference/expense/travelallowance/travelallowance-days.md]: Should be: **Required** The company's unique identifier.

File: src/api-reference/expense/travelallowance/travelallowance-days.md [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+`Context`|`string`|-|This parameter is required. For expense reports Context is `EXPENSE_REPORT`
```
Comment (sarraloew) [inline src/api-reference/expense/travelallowance/travelallowance-days.md]: Add the **Required** text to these descriptions.

File: src/api-reference/expense/travelallowance/travelallowance-days.md [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+`ContextId`|`string`|-|This parameter is required. For expense reports reportId of the report needs to be passed as contextId
```
Comment (sarraloew) [inline src/api-reference/expense/travelallowance/travelallowance-days.md]: Where do they get the report ID from? Do they need to call the Expense Report API?

File: src/api-reference/expense/travelallowance/travelallowance-days.md [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+
+### <a name="schema-one"></a>Get TravelAllowance Days Response Schema
+
+Name|Type|Format|Description
+---|---|---|---
+`ExpenseTypeCode`|`string`|-|Expense Code. eg: MEALS for TravelAllowance.
+`FullAllowanceDay`|`string`|-|Y/N. 'Y' means full allowance day and 'N' means partial allowance day.
+`AllowanceDate`|`dateTime`|[dateTime](#definition-dateTime)|TravelAllowance Date.
+`ExpenseTransactionDate`|`dateTime`|[dateTime](#definition-dateTime)|Expense transaction date.
+`ExpenseTransactionAmount`|`numeric`|-|Expense transaction amount.
```
Comment (sarraloew) [inline src/api-reference/expense/travelallowance/travelallowance-days.md:178]: Is this in the user's reimbursement currency?

File: src/api-reference/expense/travelallowance/travelallowance-days.md [hunk 10]
```diff
@@ -0,0 +1,192 @@
...
+
+* [Schema One](#schema-one)
+* [Error](#schema-error)
+
+### <a name="details-of-travelallowance-days"></a>Get Details of TravelAllowance Days
+
+#### Request
+
+```shell
+GET https://katre-report-service-banaras.us-west-2.nonprod.cnqr.deliveryapi/api/v4/
```
Comment (sarraloew) [inline src/api-reference/expense/travelallowance/travelallowance-days.md]: This is not a Concur data center URL

File: src/api-reference/expense/travelallowance/travelallowance-days.md [hunk 11]
```diff
@@ -0,0 +1,192 @@
...
+```
+
+## <a name="schema"></a>Schema
+
+### <a name="schema-one"></a>Get TravelAllowance Days Response Schema
+
+Name|Type|Format|Description
+---|---|---|---
+`ExpenseTypeCode`|`string`|-|Expense Code. eg: MEALS for TravelAllowance.
+`FullAllowanceDay`|`string`|-|Y/N. 'Y' means full allowance day and 'N' means partial allowance day.
```
Comment (sarraloew) [inline src/api-reference/expense/travelallowance/travelallowance-days.md]: This description says the values are Y/N, however the example provided has True and False. Which one is correct?

File: src/api-reference/expense/travelallowance/travelallowance-days.md [hunk 12]
```diff
@@ -0,0 +1,192 @@
...
+`AllowanceDate`|`dateTime`|[dateTime](#definition-dateTime)|TravelAllowance Date.
```
Comment (sarraloew) [inline src/api-reference/expense/travelallowance/travelallowance-days.md:176]: Is this the same as ExpenseTransactionDate? If not, what is the difference?

**Review by sarraloew — CHANGES_REQUESTED** [review]
Hi all, I've added some review questions and some requested changes. Please let me know if you have any questions.

File: src/api-reference/expense/travelallowance/travelallowance-days.md (hunk 3 above)
Comment (UdayKumarNettem) [reply src/api-reference/expense/travelallowance/travelallowance-days.md:8]: No. It's in EMEA and US

File: src/api-reference/expense/travelallowance/travelallowance-days.md (hunk 8 above)
Comment (UdayKumarNettem) [reply src/api-reference/expense/travelallowance/travelallowance-days.md]: They will call v3 API's to get the report ID.
https://developer.concur.com/api-reference/expense/expense-report/reports.html

File: src/api-reference/expense/travelallowance/travelallowance-days.md (hunk 9 above)
Comment (UdayKumarNettem) [reply src/api-reference/expense/travelallowance/travelallowance-days.md:178]: No. But the currency details can be fetched from Expense Entry API's

File: src/api-reference/expense/travelallowance/travelallowance-days.md (hunk 12 above)
Comment (UdayKumarNettem) [reply src/api-reference/expense/travelallowance/travelallowance-days.md:176]: AllowanceDate is the date of allowances in an itinerary . Expenses are created out of an allowance.Technically these are not same dates.

**Pull request closed without merging**